### ISSUE TITLE
CIFAR-10 normalize params

### ISSUE THREAD

**Issue by wufeim** [chenyaofo/pytorch-cifar-models#8]
Thanks for the pre-trained models!

I'm testing the pre-trained models on CIFAR-10 using ```torchvision.datasets.CIFAR10()``` and found that you are using different normalization params from [pytorch.org](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html):

- From [your repo](https://github.com/chenyaofo/image-classification-codebase/blob/master/conf/cifar10.conf): ```mean: [0.4914, 0.4822, 0.4465]```, ```std: [0.2023, 0.1994, 0.2010]```.
- From [pytorch.org](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html): ```transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))```.

I can reproduce the results using the same normalization params as you had. It took me some time to locate the issue so I assume clarifying the normalization params somewhere in the project README would help other people.

**Comment by chenyaofo** [comment]
Sorry for the late reply.

Actually, `mean: [0.4914, 0.4822, 0.4465], std: [0.2023, 0.1994, 0.2010]` are computed based on CIFAR-10 dataset. For more details that how to calculate `mean` and `std`, please refer to [https://discuss.pytorch.org/t/computing-the-mean-and-std-of-dataset/34949/2](https://discuss.pytorch.org/t/computing-the-mean-and-std-of-dataset/34949/2).

However, sometimes, we can not calculate `mean` and `std` because of some reasons (for example, the dataset is too large to calculate). In this case, we will use `transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))`. 

This is my understanding.

**Comment by chenyaofo** [comment]
In my opinion, these two normalized setting will yield competitive performance (I am not very sure with this).

**Comment by wufeim** [comment]
Thanks very much for the comment. For your information, I think the performance would drop by around 1% when using "the other" mean and std, which is usually significant.
